packages/core/config/irving/getServiceAliases.js

```diff
@@ -82,6 +82,17 @@ const getServiceAliases = (target) => {
     );
   }
 
+  /**
+   * Get the log service config.
+   */
+  let logServiceConfig = resolveConfigFilepath('sentry.config.js');
```

Comment:
Realized it might actually make more sense to put this in a `config/aliases.js` file in the `vip-go` package because it's only used by that package currently (assuming that's possible...I think it should be)

Reply:
I was looking at that and was going to bring it up in swarm actually! I'll give this a go.